.ruby-version

```diff
@@ -1 +1 @@
-2.7.7
+2.7.8
```

Comment:
This change was needed for the ci/cd pipeline to build correctly for a dev instance, so I think it will be needed for prod as well. We've done [similar changes](https://github.com/code-dot-org/javabuilder/commit/8106ac2e6d1880a7b270f2a4b78e10f915b0b694) in the past to keep up the the AWS version of ruby.